tests/expr/t15_while_true.test

```diff
@@ -0,0 +1,10 @@
+discard """
+  description: "
+    A `(While (Ident true))` loop never exits normally, and is thus of type
+    void
+  "
+  arguments: "c"
```

Comment:
am I missing something, why is the `arguments` necessary?

Reply:
Ah, I'll add a comment explaining why. The while loop repeats forever, which requires not running the test, otherwise the runner would never return.